src/svelte/Breadcrumbs.svelte

```diff
@@ -7,6 +7,7 @@
   const BUTTON_ELLIPSIS = Symbol();
   export let breadcrumbs: Breadcrumbs = {items: []};
   export let loadJs = true;
+  export let classNames = '';
```

Comment:
How about letting the attribute be named `class` externally?

```suggestion
  let classNames = '';
  export {classNames as class};
```